examples/example-fastmcp-mcp/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "example-fastmcp-mcp",
+  "version": "0.1.0-beta.1",
+  "description": "FastMCP Server Example (Beta) — Example implementation of an MCP server using FastMCP framework with Auth0 authentication. This example is in beta and demonstrates secure MCP server patterns with OAuth 2.0 and JWT validation.",
```

Comment:
What is the approach to move this out of beta post merging this example?

Reply:
Good call out. We are starting with beta and might not support every framework. The idea is to first validate those mcp examples and get some feedback. Framework popularity with mcp server integration is also a factor. Curious if there's an existing path we usually follow to move examples here out of beta